### ISSUE TITLE
Sentinel2: _time_report accidental reset & typo

### ISSUE THREAD

**Issue by ircwaves** [Applied-GeoSolutions/gips#222]
Funny to find things after the PR-CR is done, but they're minors:

https://github.com/Applied-GeoSolutions/gips/blob/dev/gips/data/sentinel2/sentinel2.py#L406

_time_report is a nice function.  This accidental makes me think it could be tweaked to be:
```python
def _time_report(self, message, verbosity_level=2, reset_clock=False):
```
to follow `verbose_out` style.

Also:
mis-parththesization: https://github.com/Applied-GeoSolutions/gips/blob/dev/gips/data/sentinel2/sentinel2.py#L346

**Comment by ags-tolson** [comment]
Because the time report method is stateful, changing verbosities from call to call seemed problematic, because one could change the verbosity while resetting the clock then further reports could be misleading, or even nonsensical.  So I took the liberty of adding verbosity to the state carried by the method, such that all reports after resetting the clock are at the same verbosity.

That and the fix for the erroneous call is in a commit, 5990036, in the branch '200-rad-toa'.

**Comment by ircwaves** [comment]
bueno
